### DISCUSSION TITLE
Sharing a shape

### DISCUSSION THREAD

**Discussion by nirmalkc** [TryShape/tryshape#95] [Ideas]
As a shape creator, I should be about to share the created shape in the following social media.

1. Facebook
2. Twitter
3. Instagram
4. Tumblr
5. Linkedin

**Comment by atapas** [comment]
Must need. Upvoted.

**Comment by atapas** [comment]
@nirmalkc, Do you have an initial idea to move it to as an issue for start working? Let's discuss.
